### DISCUSSION TITLE
Where can I find a list of the available `mathfont`s for PDF output ?

### DISCUSSION THREAD

**Discussion by avhz** [quarto-dev/quarto-cli#6829] [Q&A]
### Description

Question in title. Is it available somewhere ?

**Comment by mcanouil** [comment]
This is a question not related to Quarto.
Also, you'll be better of searching for that exact question on the internet using your favourite search engine, see <https://duckduckgo.com/?q=math+font+latex>.

**Comment by avhz** [comment]
This question IS related to Quarto as `mathfont` is a field in the Quarto PDF options when making a .qmd file:

https://quarto.org/docs/reference/formats/pdf.html#fonts

I took the suggested link in the docs to the fontspec package on CTAN yet cannot find a list of the possible options. 

First couple results in your passive aggressively suggested Google search is not what I am after, I know how to use things like `\mathbb{R}`, `\mathcal{F}`, etc. That is not what I need.

The third result (https://tug.org/FontCatalogue/mathfonts.html) seems to be closer to what I need, and I have indeed visited this page before, yet I cannot tell how to use these fonts in Quarto. 
If you know how, I am all ears. 

I simply want a list of possible arguments to the `mathfont` field. Does it exist?

**Reply by mcanouil** [reply]
`mathfont` is the name of a font to be used for math.

> The sans serif font family for use with xelatex or lualatex. Takes the name of any system font, using the [fontspec](https://ctan.org/pkg/fontspec) package.

The list is your "system font" list. To get that list, there are well documented methods for all operating system.

You can use whatever font available on the internet, even custom font (if sans serif font) which `fontspec` can use, if your LaTeX engine is [XETEX](https://ctan.org/pkg/xetex) or [LuaTEX](https://ctan.org/pkg/luatex) as documented.

FYI, the fourth link of the search I provided: https://tug.org/FontCatalogue/mathfonts.html

Note that the option is passed to Pandoc (see <https://pandoc.org/MANUAL.html#fonts>) which is then used by the two above mentioned LaTeX engines.

```
mathfont: texgyredejavu-math.otf 
```

```
mathfont: TeXGyreDejaVuMath-Regular
```

For short, the list of math fonts available is the list of fonts supporting math glyph installed in your LaTeX distribution and your operating system, thus there is no way to provide you with a correct list because my fonts are not necessarily yours.

PS: I did not intend to be "passive aggressive". Sorry for that!

**Comment by cscheid** [comment]
@avhz "First couple results in your passive aggressively"

We don't accept this kind of language here. Please note that there are language differences between participants here, and be willing to accommodate those. We ask you in the future to instead simply follow up with your clarification.

**Comment by eteitelbaum** [comment]
Hi, I am having a related issue in that in a template I made on a PC I specified `mathfont:` as "Cambria Math" but this does not work on a Mac. I get an error like this: 

`ERROR: 
compilation failed- no matching packages
Package fontspec Error: The font "Cambria Math" cannot be found.`

I get a similar error if I try to use "Palatino Linotype" as my main font but it is fine if I just use "Palatino." 

So maybe it is worth noting that not every font listed in the `fontspec` package or that is available on pandoc can be used in Quarto? Although I suppose @mcanouil gets at the issue when saying:  

`For short, the list of math fonts available is the list of fonts supporting math glyph installed in your LaTeX distribution and your operating system, thus there is no way to provide you with a correct list because my fonts are not necessarily yours.`

I stared at the `fontspec` documentation for about 10 minutes but it was like reading hieroglyphics. I swear that 1000 years from now there is going to be an entire discipline dedicated to unearthing the ancient secrets of LaTeX!

I asked ChatGPT for some suggestions of math fonts that would be compatible across macs and PCs and it just hallucinated some stuff. I wish there were a user-friendly guide available... Or maybe there is but I just have not seen it yet.

**Reply by mcanouil** [reply]
Welcome to $\LaTeX$ world!

Possibly, providing fonts with the document is the more reliable way to get a cross-platform document with the same font.

**Reply by eteitelbaum** [reply]
You mean provide the font as part of the extension or template? How would you do that? 

Maybe it is a question for another thread....

**Reply by mcanouil** [reply]
See the journals extensions: <https://github.com/quarto-journals>.
